### PR TITLE
null data HTML conversion

### PR DESCRIPTION
**Pull request by hoytpr**
This works on my desktop for converting the Null data table to HTML.

### CODE REVIEW

**Comment by tobyhodges** [comment]
Thanks @hoytpr it looks good on my local build too.

Reading through the related issue https://github.com/LibraryCarpentry/lc-spreadsheets/issues/124 there are some good suggestions for how the table could be made clearer and I think this is a good opportunity to make some of those changes. Specifically:

1. the literal values could be distinguished from the descriptive values in the first column. I.e. "Blank" is a description of the value that could be used and it might be helpful to adjust the table somehow to indicate that this is different from e.g. "None", which is a suggestion to use the literal value "None" to indicate missing data. My favourite option for this: format the literal values as "code" (`0`, `-`,`+`,`.`, `None`). Would you be willing to update the pull request to format the values in this way?
2. @SandiCal suggested an improvement to the explanation in the paragraph that precedes the table, and it sounded like @morskyjezek was planning to work up a pull request based on that. Would one of you have time to open up a second pull request on this repository? We could then discuss, review, and merge both here before propagating the improvements to the other two lessons that include this content.

File: _episodes/02-common-mistakes.md [hunk 1]
```diff
@@ -199,11 +266,6 @@ that are excessively long. Including the units in the field names avoids confusi
 	<td> CellType </td>
 	<td> Cell Type </td>
 </tr>
-<tr>
-	<td> Observation_01 </td>
-	<td> first_observation</td>
-	<td> 1st Obs</td>
-</tr>
```
Comment (tobyhodges) [inline _episodes/02-common-mistakes.md:206]: What is the reason for this row being removed in another table?

Comment (hoytpr) [reply _episodes/02-common-mistakes.md:206]: @tobyhodges if it was removed, it was a mistake.  I copied the other table to start a framework for the new table. I deleted an extra row when done. Hopefully I didn't actually delete that line from the other table, but it's possible. I'll certainly look in to this while considering the comments from the other maintainers.

Comment (hoytpr) [reply _episodes/02-common-mistakes.md]: I've added back the lines to the other table. Hopefully this takes care of the first issue (making the table into HTML).

**Pull request merged**